m4_sessions/1710-inning/group_four.md

```diff
@@ -1,10 +1,31 @@
-Group Four
+****Group Four****
+
+**Members**
+- David Becker
+- Jon Beckman
+- Sam Johnson
+- Yohanan Assefa
+- Michael Sagopolutele
+- Daniel Vogelsang
+
+Session Title: Our new role in social movements developers
+Description: Why should students come to this session?
 
-Session Title: 
-Description: Why should students come to this session? 
 
 Session Outcomes: What will students leave with or gain from this session?
 
+
 Session Outline:  Include the flow of the session and the time for each section.
 
-Names of group members 
+**Intro**
+
```

Comment:
Be sure to build in 10 minutes for students to have introductions.  There will be a mix of mod 1, 2, 3.